vspace/src/main/java/edu/asu/diging/vspace/web/staff/ExhibitionConfigurationController.java

```diff
@@ -0,0 +1,72 @@
+package edu.asu.diging.vspace.web.staff;
+
+import java.io.IOException;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import edu.asu.diging.vspace.core.data.ExhibitionRepository;
+import edu.asu.diging.vspace.core.data.SpaceRepository;
+import edu.asu.diging.vspace.core.factory.impl.ExhibitionFactory;
+import edu.asu.diging.vspace.core.model.ISpace;
+import edu.asu.diging.vspace.core.model.impl.Exhibition;
+import edu.asu.diging.vspace.core.model.impl.Space;
+import edu.asu.diging.vspace.core.services.impl.ExhibitionManager;
+import edu.asu.diging.vspace.core.services.impl.SpaceManager;
+
+@Controller
+public class ExhibitionConfigurationController {
+
+	@Autowired
+	private SpaceRepository spaceRepo;
+
+	@Autowired
+	private SpaceManager spaceManager;
+
+	@Autowired
+	private ExhibitionManager exhibitManager;
+
+	@Autowired
+	private ExhibitionRepository exhibitRepo;
+
+	@Autowired
+	private ExhibitionFactory exhibitFactory;
+
+	/**
+	 * @param model
+	 * @return
+	 */
```

Comment:
either your method should have proper javadoc, or no javadoc at all (in some cases this is justifiable), but this is useless